bittensor/_metagraph/__init__.py

```diff
@@ -124,7 +124,6 @@ def sync ( self, block: Optional[int] = None, lite: bool = True ) -> 'metagraph'
         self.version = torch.nn.Parameter( torch.tensor( [bittensor.__version_as_int__], dtype=torch.int64 ), requires_grad=False )
         self.block = torch.nn.Parameter( torch.tensor( subtensor.block, dtype=torch.int64 ), requires_grad=False )
         self.uids = torch.nn.Parameter( torch.tensor( [ neuron.uid for neuron in self.neurons ], dtype=torch.int64 ), requires_grad=False )
-        self.emission = torch.nn.Parameter( torch.tensor( [ neuron.emission for neuron in self.neurons ], dtype=torch.float32 ), requires_grad=False )
```

Comment:
Why is the emission removed from the metagraph?

Reply:
This was duplicated a bit lower in the metagraph